usaspending_api/api_contracts/contracts/v2/references/filter_tree/tas.md

```diff
@@ -18,6 +18,7 @@ Returns a list of toptier agencies that have at least one TAS affiliated with th
                     + `1`
                     + `2`
         0 will return only agencies, 1 will return agencies and any federal accounts under them, and so on.
+        + `filter`: `port` (optional, string) When provided, only results who's id or name matches the provided string (case insensitive) will be returned, along with any ancestors to a matching node. 
```

Comment:
we're making a renewed effort to not include any example values in the schema description